internal/cloudapi/v2/errors.go

```diff
@@ -72,69 +72,71 @@ const (
 )
 
 type ServiceErrorCode int
+type ErrorMessage string
 
 type serviceError struct {
 	code       ServiceErrorCode
 	httpStatus int
 	reason     string
+	details    string
```

Comment:
Can we make this interface{} ; to map better to the eventual type of details in the api?